docs/app/Examples/Button/Emphasis.js

```diff
@@ -0,0 +1,13 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <Button className='primary' />
+        <Button className='secondary' />
```

Comment:
Here also might be useful to have some Button content.  Either a contrived example as in the [Semantic Docs](http://semantic-ui.com/elements/button.html#emphasis) or simply "Primary" and "Secondary".